### PR TITLE
Update README.md and add installation guide

### PR DESCRIPTION
**Pull request by cdrage**
Simplifies the README.md a bit, since we not only reference user / quick
astart guides elsewhere, but some of the information is redundant.
Removes "project status".

An installation doc is also added too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,26 @@
...
-We are a very evolving project with high velocity, we have listed a [file reference specification](docs/file-reference.md) as well as document our RFC's and changes as [GitHub issues](https://github.com/kedgeproject/kedge/issues).
 
-Check out our [roadmap](ROADMAP.md) as we push towards a __0.4.0__ release.
+----
 
-## Using Kedge
+## To start using Kedge
 
-### Installing
+### Install
```
Comment (kadel) [inline README.md]: Can we link to installation.md somewhere in this section?
It includes some important information like latest builds, @pradeepto this information in readme

Comment (cdrage) [reply README.md]: I'll update it. Thanks!

Comment (cdrage) [reply README.md]: I put the latest builds in a separate installation document, but we could move this back to the README.md if you wish.

Comment (kadel) [reply README.md]: 👍 I'm fine with just linking it.

**Pull request merged**